app/src/main/java/com/poupa/vinylmusicplayer/model/Song.java

```diff
@@ -76,27 +75,6 @@ public Song(final @NonNull Song song) {
         this.year = song.year;
     }
 
-    public void setReplayGainValues(float track, float album) {
```

Comment:
ReplayGain extraction is done once with Discog - this lazy caching code is not needed any more